lib/cli/src/commands/run/mod.rs

```diff
@@ -107,11 +107,20 @@ impl Run {
             wasmer_vm::set_stack_size(self.stack_size.unwrap());
         }
 
+        // check for the preffered webc version
+        let preffered_webc = match std::env::var("WASMER_USE_WEBCV3") {
```

Comment:
This naming has a typo (two f's)... also I think we can use `preferred_webc_version` which is a bit more descriptive